src/components/search/index.tsx

```diff
@@ -89,6 +91,18 @@ const Search: FunctionComponent<SearchProps> = ({
     )
   }
 
+  const shouldDisplayLandingPageForInstructor = (
+    instructor: string[],
+    searchState: any,
+  ) => {
+    const selectedInstructor = get(
+      searchState,
+      'refinementList.instructor_name',
+      [],
+    ) as string[]
+    return isEqual(selectedInstructor, instructor)
+  }
```

Comment:
With the `shouldDisplayLandingPageForInstructor` I was able to display the curated instructor page only on the selected instructor.